apps/admin-gui/src/styles.scss

```diff
@@ -1689,3 +1676,64 @@ td.mat-mdc-cell {
 mat-icon {
   overflow: inherit !important;
 }
+
+/*************************************
+******** Customization of MDC ********
+*************************************/
+
+// Custom MDC form field
+.mdc-text-field {
+  background-color: transparent !important;
+  margin-top: 5px !important;
+}
+label:not(.mdc-floating-label--float-above) {
+  padding-top: 16px !important;
+  width: 100%;
+}
+.mat-form-field-appearance-outline {
+  label:not(.mdc-floating-label--float-above) {
+    padding-top: 0 !important;
+  }
+}
+.mdc-floating-label--float-above {
+  width: 133.33% !important;
+}
+mat-form-field mat-icon {
+  padding: 0 !important;
+}
+// TODO: Do we want to have a transparent background for focused elements?
+//.mat-focused .mat-mdc-form-field-focus-overlay {
+//  background-color: transparent !important;
+//}
+// TODO: Do we want to have left padding for labels and text in form field? (the same for hints and errors)?
+// Customization for "Items per page" form field
+//.mdc-text-field:not(.mdc-text-field--outlined) {
```

Comment:
Not sure if we need additional padding for the labels. It seems there is already some left margin.

Reply:
There is actually left padding 16px and this class instead has the aim to replace this "big" padding with just a small 2px left padding. But in my opinion, the actual padding is ok and it was just a suggestion, so I am removing the comment.